code/modules/mob/living/carbon/brain/brain.dm

```diff
@@ -54,7 +54,7 @@
 	if(in_contents_of(/obj/mecha))
 		canmove = 1
 		use_me = 1 //If it can move, let it emote
-	if(istype(loc, /obj/item/device/mmi))	canmove = 1 //mmi won't move anyways so whatever
+	else if(istype(loc, /obj/item/device/mmi))	canmove = 1 //mmi won't move anyways so whatever

```

Comment:
This is the line in particular that's causing problems: MMIs in mechs have been unable to move at all as of recent. Should I isolate this fix and put it in a separate PR?